### PR TITLE
Initial releases

### PR DESCRIPTION
**Pull request by FrankGiesecke**

### CODE REVIEW

File: 5.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+FROM php:5.3-cli
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV XDEBUG_VERSION="2.2.7"
+ENV DEBIAN_FRONTEND "noninteractive"
```
Comment (MaSpeng) [inline 5.3/Dockerfile]: ❔ I thought you prefer the style above?

File: 5.3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    && apt-get clean -yqq \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /var/cache/apk/*s
+
+SHELL ["/bin/bash", "-o", "pipefail", "-e", "-c"]
+
+# Install XDEBUG
+RUN pecl install "xdebug-${XDEBUG_VERSION}"
+
+# Remove build dependencies
+RUN apt-get remove --purge -yqq \
```
Comment (MaSpeng) [inline 5.3/Dockerfile]: ❓ The [documentation](https://wiki.ubuntuusers.de/apt/apt-get/#apt-get-remove) says that `remove` requires a minimum of one package name as argument. What does this call do?

File: 5.3/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    && apt-get --purge autoremove -yqq
```
Comment (MaSpeng) [inline 5.3/Dockerfile]: ❕ This command should do what is indented with the command above.

❓ Which style do you prefer, arguments and options or options and arguments

File: 5.4/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        sysvshm \
+        wddx \
+        xsl \
+        zip
+
+# Install XDEBUG
+RUN pecl install "xdebug-${XDEBUG_VERSION}"
+
+# Remove build dependencies
+RUN apt-get remove --purge -yqq \
```
Comment (MaSpeng) [inline 5.4/Dockerfile]: Same questions here.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,85 @@
-# docker-hub-php-cli
-Different PHP versions as CLI for CI
+# PHP CLI
+[![CircleCI](https://circleci.com/gh/final-gene/docker-hub-php-cli/tree/master.svg?style=svg)](https://circleci.com/gh/final-gene/docker-hub-php-cli/tree/master) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e067ba9a720d4b3995c21adc9182f599)](https://www.codacy.com/app/final-gene/docker-hub-php-cli?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=final-gene/docker-hub-php-cli&amp;utm_campaign=Badge_Grade)
+
+This is a image to run the [PHP CLI](http://php.net/manual/en/features.commandline.php).
+
+## Supported tags and respective Dockerfile links
+* `7.2`, `7.2-xdebug`, `latest` [(7.2/Dockerfile)](https://github.com/finalgene/docker-hub-php-cli/blob/master/7.2/Dockerfile)
```
Comment (MaSpeng) [inline README.md:7]: ❓ Why the additional tag for xdebug?

**Review by MaSpeng — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (FrankGiesecke) [reply README.md:7]: This is legacy to avoid breaking existing usages of my images.

File: 5.3/Dockerfile (hunk 2 above)
Comment (FrankGiesecke) [reply 5.3/Dockerfile]: You're right. This makes no sense and is result of `copy & paste` from an old `Dockerfile` 🤦‍♂️

**Review by MaSpeng — APPROVED** [review]

**Pull request merged**